setup.py

```diff
@@ -34,7 +34,7 @@
 from setuptools import find_packages
 
 
-VERSION = "1.4.20"
+VERSION = "1.5.0"
 
 REQS = ['click>=7.0',
         'enum-compat>=0.0.2',
```

Comment:
Can you drop this? What about pylru below?

Reply:
I should have included `future` here too, can that be removed?

Reply:
What's the deal with `pylru` ?

Reply:
There is now a built-in [LRU cache](https://docs.python.org/3/library/functools.html#functools.lru_cache), I assumed `pylru` was only for py2.7 support.